app/create/page.tsx

```diff
@@ -1,3 +1,114 @@
+"use client";
```

Comment:
I don't see why this component would need to be client side component, looks static; can you elaborate?

Linking docs about client/server components in next [here](https://nextjs.org/docs/app/building-your-application/rendering/client-components)